frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -462,6 +522,40 @@ function ReaderHighlight:addToMainMenu(menu_items)
             })
         end,
     })
+    if Screen:isColorScreen() then
```

Comment:
So, we cannot update the drawer on b/w devices? It might be useful.

Reply:
One of the issues is that the menu entry to choose the color is also hidden, so we'd need to pop quite a few of thsoe checks to allow that, and that includes @poire-z's concerns about the [fancy highlight on-press menu](https://github.com/koreader/koreader/pull/11044#discussion_r1378286993) ;).

FWIW, I imagine that, even on grayscale devices, one might want to be able to choose the color, for interoperability purposes (esp. on PDF).

But, that's a fairly gnarly can of worms to be opening ;).